flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/observer/JobObserver.java

```diff
@@ -81,14 +81,20 @@ private boolean observeFlinkJobStatus(
             flinkAppStatus.getJobStatus().setState(JOB_STATE_UNKNOWN);
             return false;
         }
-
-        updateJobStatus(flinkAppStatus.getJobStatus(), new ArrayList<>(clusterJobStatuses));
-        logger.info("Job status successfully updated");
+        String targetJobStatus =
+                updateJobStatus(flinkAppStatus.getJobStatus(), new ArrayList<>(clusterJobStatuses));
+        logger.info(
+                "Job status successfully updated from {} to {}.",
```

Comment:
To make this really nice we could add a branch:
```
if (changed) {
 "Job status successfully updated from {} to {}."
 } else {
  "Job status ({state}) unchanged."
 }
```

Reply:
Updated.